test/helpers/minitest_helper.rb

```diff
@@ -24,6 +24,8 @@
 end
 =end
 
+puts "DEM: minitest helper"
```

Comment:
We probably want to remove that comment before merging.

Reply:
done